### PR TITLE
Toilet water changes

### PR DESCRIPTION
**Pull request by JimKil3**
## What Does This PR Do
Toilet water is now somewhat toxic and does not spread in chemical smoke.

## Why It's Good For The Game
Toilet water smoke is, frankly, incredibly annoying. These changes should make it harder to use for griefing and, when it is used, more easy to justify a punishment (as it's quite literally harmful).

## Testing
Loaded in, tried to make toilet water smoke, drank toilet water

## Changelog
:cl:
tweak: Toilet water does not work in smoke reactions
tweak: Toilet water is now toxic
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/water.dm [hunk 1]
```diff
@@ -215,18 +215,21 @@
...
 
 /datum/reagent/fishwater/on_mob_life(mob/living/M)
+	var/update_flags = STATUS_UPDATE_NONE
 	if(prob(30))		// Nasty, you drank this stuff? 30% chance of the fakevomit (non-stunning version)
 		if(prob(50))	// 50/50 chance of green vomit vs normal vomit
 			M.fakevomit(1)
 		else
 			M.fakevomit(0)
-	return ..()
+	update_flags |= M.adjustToxLoss(2*REAGENTS_EFFECT_MULTIPLIER, FALSE)
```
Comment (Henri215) [inline code/modules/reagents/chemistry/reagents/water.dm:231]:
```suggestion
	update_flags |= M.adjustToxLoss(2 * REAGENTS_EFFECT_MULTIPLIER, FALSE)
```

**Review by Contrabang — APPROVED** [review]
lgtm

**Comment by Mitchs98** [comment]
Finally, tiders that run around drinking toilet water to spam puke will learn the consequences of their stupidity.

**Comment by spacegeese** [comment]
how am i gonna fucking fart now??

**Comment by ItsMarmite** [comment]
You're aware this makes toilet water a VERY potent chemical for use in syringe guns right?

**Comment by Varelion** [comment]
Personally, I hope this PR doesn't go through. Harmless yet annoying tiding sources seems important to have; a way for rascals to do their thing without bwoinking.

**Comment by DGamerL** [comment]
> Personally, I hope this PR doesn't go through. Harmless yet annoying tiding sources seems important to have; a way for rascals to do their thing without bwoinking.

If we look at it from a RP perspective - which is good to keep in mind, as we are a MRP server - puking everywhere is a very weird thing to do. Imagine you're at work and some guy just starts puking on the ground "as a joke". I can guarantee you that if he keeps doing that every few days, he won't be working there for long.

**Comment by JimKil3** [comment]
Updated to only do 1 toxin/cycle in the latest commit

**Comment by Mitchs98** [comment]
> Personally, I hope this PR doesn't go through. Harmless yet annoying tiding sources seems important to have; a way for rascals to do their thing without bwoinking.

Tiding shouldn't be encouraged, really, especially when you genuinely only are doing it to annoy or inconvience others. The absolute only thing toilet water and toilet water grenades do is annoy the majority of people affected by it.

As DG said we're an MRP server. This falls squarely in the realm of purely LRP as well.

**Comment by Varelion** [comment]
> > Personally, I hope this PR doesn't go through. Harmless yet annoying tiding sources seems important to have; a way for rascals to do their thing without bwoinking.
> 
> Tiding shouldn't be encouraged, really, especially when you genuinely only are doing it to annoy or inconvience others. The absolute only thing toilet water and toilet water grenades do is annoy the majority of people affected by it.
> 
> As DG said we're an MRP server. This falls squarely in the realm of purely LRP as well.

If that were true why is clown even a role? Or mime?

**Comment by Mitchs98** [comment]
> > > Personally, I hope this PR doesn't go through. Harmless yet annoying tiding sources seems important to have; a way for rascals to do their thing without bwoinking.
> > 
> > 
> > Tiding shouldn't be encouraged, really, especially when you genuinely only are doing it to annoy or inconvience others. The absolute only thing toilet water and toilet water grenades do is annoy the majority of people affected by it.
> > As DG said we're an MRP server. This falls squarely in the realm of purely LRP as well.
> 
> If that were true why is clown even a role? Or mime?

People causing problems in those roles are player issues, and they are _roles_ not mechanics. There are plenty of good clowns and mimes, with actually quite a few more on the rise lately. Clowns and Mimes can be good for the round, have real gimmicks, and actually be funny and entertaining.

Toilet water can do none of these things.

**Comment by spacegeese** [comment]
couldn't you just make the easily accessible toxin not vomit people instead? sure it might not make sense but so does just disabling smoke for this specific chemical altogether because of some goofball with a grenade

**Comment by Varelion** [comment]
> > > > Personally, I hope this PR doesn't go through. Harmless yet annoying tiding sources seems important to have; a way for rascals to do their thing without bwoinking.
> > > 
> > > 
> > > Tiding shouldn't be encouraged, really, especially when you genuinely only are doing it to annoy or inconvience others. The absolute only thing toilet water and toilet water grenades do is annoy the majority of people affected by it.
> > > As DG said we're an MRP server. This falls squarely in the realm of purely LRP as well.
> > 
> > 
> > If that were true why is clown even a role? Or mime?
> 
> People causing problems in those roles are player issues, and they are _roles_ not mechanics. There are plenty of good clowns and mimes, with actually quite a few more on the rise lately. Clowns and Mimes can be good for the round, have real gimmicks, and actually be funny and entertaining.
> 
> Toilet water can do none of these things.

I disagree. I think people overreacting to the throw up is intensely amusing.

**Comment by Mitchs98** [comment]
> > > > > Personally, I hope this PR doesn't go through. Harmless yet annoying tiding sources seems important to have; a way for rascals to do their thing without bwoinking.
> > > > 
> > > > 
> > > > Tiding shouldn't be encouraged, really, especially when you genuinely only are doing it to annoy or inconvience others. The absolute only thing toilet water and toilet water grenades do is annoy the majority of people affected by it.
> > > > As DG said we're an MRP server. This falls squarely in the realm of purely LRP as well.
> > > 
> > > 
> > > If that were true why is clown even a role? Or mime?
> > 
> > 
> > People causing problems in those roles are player issues, and they are _roles_ not mechanics. There are plenty of good clowns and mimes, with actually quite a few more on the rise lately. Clowns and Mimes can be good for the round, have real gimmicks, and actually be funny and entertaining.
> > Toilet water can do none of these things.
> 
> I disagree. I think people overreacting to the throw up is intensely amusing.

This honestly just proves my point and reasoning for agreeing with the PR.

**Review by S34NW — APPROVED** [review]

**Pull request merged**